prime-router/settings/staging/0170-epic-staging-sender-receiver.yml

```diff
@@ -0,0 +1,115 @@
+# To submit this to localhost, run:
+# ./prime multiple-settings set --env staging --input ./settings/staging/0170-epic-staging-sender-receiver.yml
+#
+# Description:
+#   These following setting are for Epic Systems Integration Testing
+#   Order sender (epic.etor-nbs-orders)
+#   Result Receiver (epic.etor-nbs-results)
+#
+---
+- name: "epic"
+  description: "Epic Systems Integration Testing"
+  jurisdiction: "STATE"
+  stateCode: LA
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders:
+    - !<full-elr>
+      name: "etor-nbs-orders"
+      organizationName: "epic"
+      format: "HL7"
+      customerStatus: "active"
+      schemaName: ""
+      processingType: "async"
+      allowDuplicates: true
+      senderType: null
+      primarySubmissionMethod: null
+      topic: "etor-ti"
+      version: null
+      createdBy: null
+      createdAt: null
+  receivers:
+    - name: "etor-nbs-results"
+      organizationName: "epic"
+      topic: "etor-ti"
+      customerStatus: "active"
+      translation: !<HL7>
+        schemaName: "classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: false
+        receivingApplicationName: "EPIC"
+        receivingApplicationOID: null
+        receivingFacilityName: "LA"
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: {}
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "standard"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(MessageHeader).event.code = 'R01')"
+        - "(%performerState.exists() and %performerState = 'LA') or (%patientState.exists() and %patientState = 'LA')"
```

Comment:
Hey @oslynn -- What HL7 field is performerState mapped to? My understanding is -- for and ETOR receiver -- we'll need to route based on MSH-5 or MSH-6.

Also, my understanding is patientState (assuming this is from the address in PID) should never be used for newborn screening. Example to help clarify:
- Mary lives in NY, but gives birth while on vacation in New Orleans, LA.
- The New Orleans hospital is required to submit a newborn screening specimen to the LA public health lab, regardless of Mary's state of residence.
- The results message that's sent a few days later will then need to be returned to the New Orleans hospital. The hospital will later forwarded it on to Mary's baby's pediatrician in NY.

Reply:
Ah, every good catch, THANK YOU.  You are correct, we need to route according to MSH-5 and MSH-6.  In your above example is indeed, correct.  I will make the change.